### PR TITLE
fix(eventing): add passing events_url when eventing is enabled

### PR DESCRIPTION
**Pull request by sinhaashish**

### CODE REVIEW

File: chart/templates/mayastor/agents/core/agent-core-deployment.yaml [hunk 1]
```diff
@@ -55,7 +55,8 @@ spec:
             - "--snapshot-commitment={{ .Values.agents.core.capacity.thin.snapshotCommitment }}"
             - "--volume-commitment-initial={{ .Values.agents.core.capacity.thin.volumeCommitmentInitial }}"
             - "--volume-commitment={{ .Values.agents.core.capacity.thin.volumeCommitment }}"{{ if .Values.agents.core.partialRebuildWaitPeriod }}
-            - "--faulted-child-wait-period={{ .Values.agents.core.partialRebuildWaitPeriod }}"{{ end }}
+            - "--faulted-child-wait-period={{ .Values.agents.core.partialRebuildWaitPeriod }}"{{ end }}{{ if .Values.eventing.enabled }}
+            - "--mbus-url=nats://mayastor-nats:4222"{{ end }}
```
Comment (datacore-vvarakantham) [inline chart/templates/mayastor/agents/core/agent-core-deployment.yaml:59]: The arg should be events-url.

**Review by hrudaya21 — APPROVED** [review]

File: chart/templates/mayastor/agents/core/agent-core-deployment.yaml (hunk 1 above)
Comment (sinhaashish) [reply chart/templates/mayastor/agents/core/agent-core-deployment.yaml]: done

**Review by datacore-vvarakantham — APPROVED** [review]
LGTM

**Comment by sinhaashish** [comment]
bors merge

File: chart/templates/mayastor/agents/core/agent-core-deployment.yaml [hunk 2]
```diff
@@ -55,7 +55,8 @@ spec:
             - "--snapshot-commitment={{ .Values.agents.core.capacity.thin.snapshotCommitment }}"
             - "--volume-commitment-initial={{ .Values.agents.core.capacity.thin.volumeCommitmentInitial }}"
             - "--volume-commitment={{ .Values.agents.core.capacity.thin.volumeCommitment }}"{{ if .Values.agents.core.partialRebuildWaitPeriod }}
-            - "--faulted-child-wait-period={{ .Values.agents.core.partialRebuildWaitPeriod }}"{{ end }}
+            - "--faulted-child-wait-period={{ .Values.agents.core.partialRebuildWaitPeriod }}"{{ end }}{{ if .Values.eventing.enabled }}
+            - "--events-url=nats://mayastor-nats:4222"{{ end }}
```
Comment (tiagolobocastro) [inline chart/templates/mayastor/agents/core/agent-core-deployment.yaml:59]: This is not correct...
Should be `nats://{{ .Release.Name }}-nats:4222`

**Pull request merged**